packages/code-du-travail-frontend/pages/outils/index.js

```diff
@@ -29,15 +29,16 @@ const Outils = ({ pageUrl, ogImage, emailTemplates }) => (
       <Container>
         <PageTitle>Retrouvez tous nos outils</PageTitle>
         <CardList title="Nos outils de calcul">
-          {outils
-            .filter(({ href }) => href.startsWith("/outils"))
-            .map(({ title, hrefTitle, button, slug, href }) => (
-              <Link href={href} as={slug} passHref key={slug}>
-                <Tile button={button} title={hrefTitle}>
-                  {title}
-                </Tile>
-              </Link>
-            ))}
+          {tools.map(({ title, slug }) => (
+            <Link
+              href={`/${getRouteBySource(SOURCES.TOOLS)}/[slug]`}
+              as={`/${getRouteBySource(SOURCES.TOOLS)}/${slug}`}
+              passHref
+              key={slug}
+            >
+              <Tile button={"Démarrer"}>{title}</Tile>
```

Comment:
Since action are named according the document "source", I would prefer add a function here so we can use it every where without worrying making a typo
```js
getToolsCTALabel(source) {
  switch(source): 
    case SOURCES.TOOLS: return "Démarrer"
    case SOURCES.LETTERS: return "Consulter"
    default: return "Voir"
}